dataframe/dataframe.go

```diff
@@ -581,6 +595,8 @@ func (df DataFrame) Rapply(f func(series.Series) series.Series) DataFrame {
 			return series.Float
 		case hasInts:
 			return series.Int
+		case hasTimes:
```

Comment:
This case has to appear just below hasBool so that the priority is:

- String
- Bool
- Time
- Float
- Int